js_modules/dagit/packages/ui/src/components/Suggest.tsx

```diff
@@ -65,5 +68,30 @@ export const SuggestWIP = <T,>(props: React.PropsWithChildren<SuggestProps<T>>)
     className: 'dagit-suggest-input',
   };
 
-  return <BlueprintSuggest {...props} inputProps={inputProps} popoverProps={popoverProps} />;
+  return (
+    <BlueprintSuggest<T>
+      {...props}
+      inputProps={inputProps}
+      itemListRenderer={(props) => (
+        <List
+          style={{outline: 'none', marginRight: -5, paddingRight: 5}}
+          rowCount={props.filteredItems.length}
+          scrollToIndex={
+            props.activeItem && !isCreateNewItem(props.activeItem)
+              ? props.filteredItems.indexOf(props.activeItem)
+              : undefined
+          }
+          rowHeight={MENU_ITEM_HEIGHT}
+          rowRenderer={(a) => (
+            <div key={a.index} style={a.style}>
+              {props.renderItem(props.filteredItems[a.index] as T, a.index)}
+            </div>
+          )}
+          width={250}
+          height={Math.min(props.filteredItems.length * MENU_ITEM_HEIGHT, 250)}
```

Comment:
where does `250` come from ?

Reply:
Ahh I'll pull this out to a constant in the file, I think this is just a random value that looks nice but we can make it more explicit.